docs/core/preview3/tools/dotnet-add-package.md

```diff
@@ -0,0 +1,81 @@
+---
+title: dotnet-add package command | Microsoft Docs
+description: The dotnet-add package command provides a convenient option to add NuGet package reference to a project.
+keywords: dotnet-add , CLI, CLI command, .NET Core
+author: spboyer
+ms.author: mairaw
+ms.date: 02/16/2017
+ms.topic: article
+ms.prod: .net-core
+ms.technology: dotnet-cli
+ms.devlang: dotnet
+ms.assetid: 88e0da69-a5ea-46cc-8b46-5493242b7af9
+---
+# dotnet-add package
+
+[!INCLUDE[preview-warning](../../../includes/warning.md)]
+
+## Name
+
+`dotnet-add package` - Add package reference to a project file.
+
+## Synopsis
+
+```
+dotnet add [<PROJECT>] package <PACKAGE_NAME> [-v|--version] [-f|--framework] [-s|--source] [--package-directory] [-h|--help]
+dotnet add package [-h|--help]
+```
+
+## Description
+
+The `dotnet add package` command provides a convenient option to add a NuGet package reference to a project.
+
+## Arguments
+
+`PROJECT`
+
+Project name to use. If not specified, the command will search the current directory for one.
+
+`PACKAGE_NAME`
+
+Package reference to add.
+
+## Options
+
+`-h|--help`
+
+Show help information for the command and/or argument
+
+`-v|--version`
```

Comment:
If you look at our other command pages, you'll see that I usually add the option argument too `-v|--version <PACKAGE_VERSION>`. Since we're not restricted in size, I don't use the exact string shown in --help